lib/api/controllers/serverController.js

```diff
@@ -110,6 +112,28 @@ class ServerController extends NativeController {
     return config;
   }
 
+  /**
+   * Returns the Kuzzle capabilities
+   * @returns {Promise<Object>}
+   */
+  async capabilities () {
+    const config = JSON.parse(JSON.stringify(global.kuzzle.config));
+    const publicApi = await this.publicApi();
+    const services = [];
+
+    _.map(config.services, (val, key) => {
```

Comment:
In general, avoid usage of lodash methods, here you could use `Object.entries(config.services).map`

Also, the purpose of `map` is to create an array from another array. Here an useless array is created and returned by `map` because you are populating the array you created l122.

Finally, we tend to avoid the usage of `map` when it's not needed because the `for (const ... of ...)` is more readable.